src/core/agent/services/multiSigService.ts

```diff
@@ -530,6 +530,17 @@ class MultiSigService extends AgentService {
     return { done: false };
   }
 
+  @OnlineOnly
+  async rotateLocalMember(ourIdentifier: string) {
```

Comment:
We use `ourIdentifier` in the code to indicate the local member - this parameter would be better called `(multisigId: string)`